README.md

```diff
@@ -1,5 +1,5 @@
 # aedes-packet
-[![Build Status](https://travis-ci.org/mcollina/aedes-packet.svg?branch=master)](https://travis-ci.org/mcollina/aedes-packet)
+![](https://github.com/moscajs/aedes-packet/workflows/ci/badge.svg)
 [![Dependencies Status](https://david-dm.org/mcollina/aedes-packet/status.svg)](https://david-dm.org/mcollina/aedes-packet)
```

Comment:
You forgot to fix the link here

Reply:
I leave it for another PR https://github.com/moscajs/aedes-packet/pull/8